### ISSUE TITLE
Zoobank - scientificNameAuthorship empty values filled with the string: <Unspecified Agent>

### ISSUE THREAD

**Issue by camiplata** [CatalogueOfLife/data#711]
dwc:scientificNameAuthorship has empty values as: "< Unspecified Agent >", 46902 records (11%) are affected by it, 43299 genus, 173 species, 1 subspecies.

See: https://www.checklistbank.org/dataset/2037/taxon/bbb21db2-66ac-47ea-836f-805854c026d8

**Comment by deepreef** [comment]
Yes! But... we're still in the process of migrating our web server to a much more robust infrastructure.  This was supposed to have been completed last year, but there have been delays.  The IPT needs to be re-established on the server that currently holds the live database.  It's on the to-do list, but not sure exactly when it will happen.  I will make sure the "<Unspecified Agent> is removed from Authorship in the next IPT export -- just not sure when that will happen.

**Comment by camiplata** [comment]
Thanks for your reply @deepreef. All the best for the migration

**Comment by camiplata** [comment]
The names with this problem are almost gone on the Zoobank version from 2026-08-29 Now we have some Unspecified Author, but we can deal with these in a new issue same with the "deleted" ones.

**Comment by mdoering** [comment]
Lets please patch the metadata and change the title to just ZooBank:
https://www.checklistbank.org/dataset/316276/source/2037
